keylime-agent/src/crypto.rs

```diff
@@ -44,6 +43,15 @@ pub(crate) fn load_x509(input_cert_path: &Path) -> Result<X509> {
     Ok(cert)
 }
 
+pub(crate) fn load_x509_cert_chain(
+    input_cert_path: &Path,
+) -> Result<Vec<X509>> {
+    let contents = read_to_string(input_cert_path)?;
+    let mut cert_chain = X509::stack_from_pem(contents.as_bytes())?;
```

Comment:
you are not mutating cert_chain, and if I read the code correcty this can be the last line:

```rust
X509::stack_from_pem(contents.as_bytes())
```

Reply:
Yes fixed. I forgot to change that, thanks for catching it!